Completed 'Hello Microverse' exercise

astroboyReloaded opened the pull request:
# Steps taken during this exercise

- Installed `Lighthouse` and `Webhint` linters.
- Created the `.gitignore` file and added all files that need to be ignored.
- Created the `index.html` file and added an H1 with the 'Hello Microverse' text.
- Created the `style.css` file and included basic styling.
- Created a `.github/workflows` folder and included a `linters.yml` file.

brainconnect93 commented on .gitignore:
```diff
@@ -0,0 +1,6 @@
+# .gitignore
+
+node_modules/
+.stylelintrc.json
+.hintrc
```
- [ ] I would suggest we strictly follow the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md
). In the requirement we are requested to ignore only our ```node_modules``` and the ```test.md``` file. Kindly remove the .stylelintrc.json and .hintrc files from gitignore and let it be in your root directory as shown in the [Linters file](https://github.com/microverseinc/linters-config/tree/master/html-css)

brainconnect93 commented on index.html:
```diff
@@ -0,0 +1,13 @@
+    <title>Hello Microverse</title>
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
```
- [ ] In the project requirement, It was stated that we add a ```class``` to our header and use the class name to style our header.
for example: 👇 
```
<h1 class="head">Hello Microverse!</h1>
```

brainconnect93 commented on style.css:
```diff
@@ -0,0 +1,7 @@
+  text-align: center;
+}
+
+h1 {
```
- [ ] Kindly make use of the ```class``` name created in the HTML header to style your HTML content instead of using an ```h1``` tag for the styling
for example: 👇 
```CSS
.head {
  color: green;
}
```

brainconnect93 reviewed (changes requested):
Hi @astroboyReloaded,

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966">

## Highlights
- Good use of GitHub flow 👌
- Linters set up well and passes  🟢
- Commit messages are descriptive and meaningful 🚀

## Required Changes ♻️
_Check the comments under the review._

## Optional changes
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

astroboyReloaded commented:
**Ok! Thank you for your comments!! I've made the requested changes:**
- Removed `.hintrc` and `.stylelintrc.json` from the .gitignore.
- Created a class for the H1 element and used it as the selector to style it.
- Added the `.hintrc` and `.stylelintrc.json` files to the repo to keep track of them.

ridwanediallo commented on index.html:
```diff
@@ -0,0 +1,13 @@
+    <title>Hello Microverse</title>
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
```
- [ ] Please as requested by the previous reviewer add a class to your `h1` and use that class in your `style.css` file to style it.
#### You can use the example he provided you   👇️
`<h1 class="heading">Hello Microverse!</h1>`
#### And you can do this  👇️ in your `style.css` file 
```
.heading {
  color: dodgerblue;
}
```

astroboyReloaded replied:
# OMG!

__I must not have saved it correctly_... Ok, it's done!

ridwanediallo reviewed (changes requested):
# Changes Required ♻️
## Hi @astroboyReloaded , 


![AlmostThereReachGIF](https://user-images.githubusercontent.com/55918904/191301799-36d07166-0e35-4582-a231-7a2d780f7783.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Owusu-Desmond reviewed (approved):
Hi @astroboyReloaded , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media2.giphy.com/media/26gsvAm8UPaczzXz2/giphy.gif"/>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
